integration-tests/mockApis/prisonApi.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
         },
         jsonBody: {
           description: 'ASSO A Wing',
```

Comment:
could maybe make this more realistic - to we get an all upper caps name usually for description?

Reply:
yes, description is all upper case, UserDescription is title case

Reply:
so probably better to make it more realistic and make description uppercase - the tests should still pass as we use the other but this would provide an extra check

Reply:
will do. 
Some tests will fail because they are expecting mixed case but I think there are only two of them